CampaignConfig/fclless/mdc2024_primary.cfg

```diff
@@ -0,0 +1,119 @@
+[global]
+includes = /exp/mu2e/app/users/sophie/ProdClone/Production/CampaignConfig/mdc2024_prolog.cfg
+run_number = 1210
```

Comment:
A different run number is set in gen_primary.    We only recently identified a case where run numbers are needed in production, to distinguish Extracted from normal geometry sims.  This requires consistency in all the scripts.